C24_WMDE_Desktop_EN_02 (round 1)

moiikana opened the pull request:
- based on en 01
- var: text change
- round 1, went live on 15th of October

- round 2 of this banner: https://github.com/wmde/fundraising-banners/pull/582

Ticket: https://phabricator.wikimedia.org/T374101

moiikana committed:
Prepare campaign for C24_WMDE_Desktop_EN_02

- based on en 01
- Change start date to 241015

Ticket: https://phabricator.wikimedia.org/T374101

moiikana committed:
Prepare CTRL for C24_WMDE_Desktop_EN_02

- base on EN01 ctrl

https://phabricator.wikimedia.org/T374101

moiikana committed:
Implement VAR for C24_WMDE_Desktop_EN_02

- text change
- add style support for the change in https://github.com/wmde/fundraising-banners/pull/579

https://phabricator.wikimedia.org/T374101

- Make payment icons work in dark and light mode
 -> this change was needed because this PR got rebased https://github.com/wmde/fundraising-banners/pull/579

Ticket: https://phabricator.wikimedia.org/T374101

Abban commented on banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss:
```diff
@@ -0,0 +1,57 @@
+// This is the file where we import the theme-specific component styles
+@use 'src/themes/Treedip/swatches/skin_vector-2022' with (
```
When https://github.com/wmde/fundraising-banners/pull/579 is merged the payment icons here will break. You should be able to add `$select-group-image-label: true,` to the list already to stop that.

moiikana replied:
added that line

Abban replied:
Oh, I expected SASS would ignore an undefined parameter as the IDE doesn't throw warnings, but apparently the compiler doesn't like it.

moiikana replied:
removed the line and will add it back later, when #579 is actually merged

moiikana commented:
rebased main for "is late progress" feature

moiikana committed:
Add UoF changes for infographics

- changes from this PR https://github.com/wmde/fundraising-banners/pull/563
- update content repo

moiikana commented:
contains the latest UoF changes from https://github.com/wmde/fundraising-banners/pull/563 :heavy_check_mark: 
- [x] banner ctrl
- [x] banner var
- [x] fallback banner

Sperling-0 commented:
Already done:

⚠️ Rebase this PR from latest main.

To get the latest UoF working:

- For Desktop banners: `BannerCtrl.vue`, `BannerVar.vue`, `FallbackBanner.vue` needs to be updated. (If it is DE desktop include DE infographic and vice versa)
- For Mobile/ Ipad banners: `BannerCtrl.vue`, `BannerVar.vue` needs to be updated. (If it is DE mobile include DE infographic and vice versa)

moiikana commented:
is rebased and contains latest changes necessary for https://github.com/wmde/fundraising-banners/pull/579
